package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "4.0.0",
+	"version": "4.1.0-beta.1",
```

Comment:
@romario-0 Please note that the Dev console migration epic is already published as [a 4.1.0-alpha](https://github.com/adobe/aio-cli-plugin-api-mesh/pull/195/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3). So would be a good thing to work with @dani97 on this package version conflict